integreat_cms/cms/views/feedback/region_feedback_actions.py

```diff
@@ -204,8 +208,13 @@ def export_region_feedback(
         data = resource.export_resource(obj)
         dataset.append(data)
 
-    response = HttpResponse(dataset.csv, content_type="text/csv")
+    if file_format in map(lambda f: f.title, format_registry.formats()):
```

Comment:
```suggestion
    if file_format in (f.title for f in format_registry.formats()):
```
Since we recently had a pr removing all usages of map, I think we shouldn't use it anymore